lib/grokngelf/importers/dynflow_tasks.rb

```diff
@@ -0,0 +1,196 @@
+        @tasks_dump = File.join(log, 'foreman_tasks_tasks.csv')
+        @actions_dump = File.join(log, 'dynflow_actions.csv')
+
+        create_database
+        load_execution_plans(@execution_plans_dump)
+        load_actions(@actions_dump)
+        load_steps(@steps_dump)
+        load_foreman_tasks_tasks(@tasks_dump)
+        import_execution_plans
+        import_steps
+      end
+
+      def load_execution_plans(dump)
+        puts "Loading Dynflow Execution plans from [#{dump}]..."
+        CSV.read(dump, headers: false).each do |row|
+          @db.execute("insert into dynflow_execution_plans values ( ?, ?, ?, ?, ?, ?, ?, ?)", row)
+        end
+      end
+
+      def load_actions(dump)
+        puts "Loading Dynflow actions from [#{dump}]..."
+        CSV.read(dump, headers: false).each do |row|
+          @db.execute("insert into dynflow_actions values ( ?, ?, ?, ?, ?)", row)
+        end
+      end
+
+      def load_steps(dump)
+        puts "Loading Dynflow steps from [#{dump}]..."
+        CSV.read(dump, headers: false).each do |row|
+          @db.execute("insert into dynflow_steps values ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", row)
+        end
+      end
+
+      def load_foreman_tasks_tasks(dump)
+        puts "Loading Foreman tasks from [#{dump}]..."
+        CSV.read(dump, headers: false).each do |row|
+          @db.execute("insert into foreman_tasks_tasks values ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", row)
+        end
+      end
+
+      def import_execution_plans
+        @db.execute("select ep.uuid, a.data, ep.state, ep.result, ep.started_at, ep.ended_at, ep.real_time, ep.execution_time,
+              ftt.id, ftt.type, ftt.label, ftt.start_at, ftt.start_before
+            from dynflow_execution_plans ep left join foreman_tasks_tasks ftt on ep.uuid = ftt.external_id, dynflow_actions a
+            where ep.uuid = a.execution_plan_uuid and a.id = 1" ).each do |row|
+          plan_data = JSON.parse(row[1])
+          data = {
+            'timestamp' => DateTime.strptime(row[4], '%Y-%m-%d %H:%M:%S').to_time.to_i,
+            'level' => helpers.level(:INFO),
+            'level_hr' => 'INFO',
+            'program' => 'dynflow',
+            'short_message' => "Execution plan #{row[0]} started",
+            'dynflow_object' => 'execution_plan',
+            'dynflow_plan_id' => row[0],
+            'dynflow_class' => plan_data['class'],
+            'dynflow_input' => plan_data['input'],
+            'dynflow_output' => plan_data['output'],
+            'dynflow_event_type' => 'start',
+            'dynflow_state' => row[2],
+            'dynflow_result' => row[3],
+            'dynflow_started_at' => row[4],
+            'dynflow_ended_at' => row[5],
+            'dynflow_real_time' => row[6].to_f,
+            'dynflow_real_time_hr' => helpers.humanize_elapsed_time(row[6].to_i),
+            'dynflow_execution_time' => row[7].to_f,
+            'foreman_task_id' => row[8],
+            'foreman_task_type' => row[9],
```

Comment:
this is not useful for parsing (at lest at thie moment, as there is only one kind of task)